src/models/utils.ts

```diff
@@ -254,7 +255,9 @@ export function isSameConnection(conn: interfaces.IConnectionCredentials, expect
         expectedConn.server === conn.server
         && isSameDatabase(expectedConn.database, conn.database)
         && isSameAuthenticationType(expectedConn.authenticationType, conn.authenticationType)
-        && expectedConn.user === conn.user
+        && (conn.authenticationType === LocalizedConstants.authTypeSql ?
```

Comment:
>authenticationType [](start = 17, length = 18)

The authenticationType stored in the profile shouldn't be localized. If it is I'd consider that a bug